Formula/gobject-introspection.rb

```diff
@@ -23,8 +23,8 @@ class GobjectIntrospection < Formula
   depends_on "cairo"
   depends_on "glib"
   depends_on "pkg-config"
-  # Ships a `_giscanner.cpython-310-darwin.so`, so needs a specific version.
-  depends_on "python@3.10"
+  # Ships a `_giscanner.cpython-311-darwin.so`, so needs a specific version.
+  depends_on "python@3.11"
```

Comment:
Not sure if this impacts any dependents? Might just be for `gobject-introspections` own scripts/commands.